src/widgets/settingspages/AliasesPage.cpp

```diff
@@ -0,0 +1,45 @@
+#include "AliasesPage.hpp"
+
+#include "controllers/aliases/AliasesModel.hpp"
+#include "singletons/Settings.hpp"
+#include "singletons/WindowManager.hpp"
+#include "util/LayoutCreator.hpp"
+#include "widgets/Window.hpp"
+#include "widgets/helper/EditableModelView.hpp"
+
+#include <QTableView>
+
+#include <QHeaderView>
+
+namespace chatterino {
+
+AliasesPage::AliasesPage()
+{
+    LayoutCreator<AliasesPage> layoutCreator(this);
+    auto layout = layoutCreator.setLayoutType<QVBoxLayout>();
+
+    layout.emplace<QLabel>("Gives an alias to a user.");
```

Comment:
```suggestion
    layout.emplace<QLabel>("Give an alias to a user.");
```
Just to be consistent with imperative verbs in other places (such as highlights, filters, etc)